packages/backend/src/database/migrations/20230809132300_add_google_oauth_token.ts

```diff
@@ -0,0 +1,15 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable('openid_identities', (t) => {
+        t.string('refresh_token').nullable();
```

Comment:
if you want this to have no limit it should be `.text` 

`.string` is equal to `VARCHAR(255)` so it'll error if there are more than 255 characters